server/src/routers/api/profileRouter.js

```diff
@@ -32,4 +33,28 @@ router.get("/:id", async(req,res)=> {
     }
 })
 
+
+router.post("/me" , auth, async (req, res) => {
+  req.body.userField = req.user._id;
+  const profile = new UserProfile(req.body)
+  try{
+      await profile.populate().save();
+      res.status(201).json({msg: "Your account has been created successfully",profile})
+  }catch(e){
+      console.log(e);
+      res.status(400).send({ errors: [{msg: e.message}] });
+  } 
+})
+
+router.delete('/me', auth, async(req,res)=>{
+  try{
+      await UserProfile.deleteOne({userField: req.user._id})
+      await req.user.remove()
+      res.send(req.user);
+  }catch(e){
+      res.status(400).send(e);
+  }
+})
+
+

```

Comment:
Attach screenshots.

Reply:
![image](https://user-images.githubusercontent.com/57807670/99551522-f2a1c300-29e1-11eb-8a79-34aab4331f30.png)

Reply:
In the route your have responded as follows:-
res.send(req.user);

But the screenshot has a completely diff format.

Do the following changes Today itself :- 
1. Undo all the changes made in the auth.js file
2. The response format in the screenshot is correct, make that change in the route.

Reply:
@Codess08 I was doing a small mistake in postman. I have fixed the code now. Had to change this line " _id: decoded._id " in the auth file though.